README.md

```diff
@@ -14,11 +14,20 @@ Lint markdown files using [remark-lint][remark-lint] and the
 
 If there is no configuration found for **remark-lint**, this linter runs [remark-preset-lint-consistent][consistent] and
 [remark-preset-lint-recommended][recommended] (both can be turned off).
+You can a also turn on [remark-preset-lint-markdown-style-guide][styleguide].
 
 If there *is* configuration for **remark-lint**, through `.remarkrc` files
 or `remarkConfig` in `package.json`s, this linter works just like
-[remark-cli][cli] but only uses the **remark-lint** plugin. Make sure
-**remark-lint** is installed in this case (locally).
+[remark-cli][cli].
+
+You should probably install modules referenced in `.remarkrc` files locally
+(`npm install` without the `-g` or `--global` flag). If you do install modules
+globally, you must either use [nvm][], or have
```

Comment:
❓  Using `nvm` with Atom is tricky, you need to launch Atom from the environment where it is active. I _think_ on macOS and *nix platforms this can be done in a per-window basis, but on Windows _all_ instances of Atom need to be closed to switch Node.js versions.

Should this be noted in the readme?